synthtool/gcp/templates/java_library/CONTRIBUTING.md

```diff
@@ -25,4 +25,106 @@ information on using pull requests.
 ## Community Guidelines
 
 This project follows
-[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
\ No newline at end of file
+[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
+
+## Building the project
+
+To build, package, and run all unit tests simply run the command
```

Comment:
When users try to follow instructions and fail, reading the word "simply" can make the experience feel even more frustrating.  Can we simply remove the word "simply?"